code/game/objects/items/storage/medkit.dm

```diff
@@ -320,6 +320,56 @@
 	)
 	generate_items_inside(items_inside,src)
 
+/obj/item/storage/medkit/coroner
+	name = "compact coroner's medkit"
+	desc = "a smaller medical kit designed primarily for assisting in dissecting the deceased, rather than treating the living."
```

Comment:
capital A at the start of both descriptions for the medkits

Reply:
got these, thanks!